db/schema.rb

```diff
@@ -15,7 +15,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "hstore"
```

Comment:
apoi adaugi linia asta la loc in schema si apoi nu o sa o mai scoata daca ai adaugat hstore